.buildkite/pipeline.yml

```diff
@@ -138,21 +138,20 @@ steps:
   - label: ":jenkins: Release - Package Registry Distribution"
     key: "release-package-registry"
     trigger: "package-registry-release-package-registry-distribution"
+    async: true
     build:
       branch: "main"
       meta_data:
         DOCKER_TAG: "${BUILDKITE_TAG}"
     if: "build.env('BUILDKITE_TAG') != ''"
-    depends_on:
-      - step: "release-test"
-        allow_failure: false
```

Comment:
Allow to start this trigger step just when the pipeline starts. No dependencies.

Reply:
If I'm not wrong in Jenkins all steps were running one by one, why did you decide to change the dependencies?
Are you sure if this sequence doesn't affect the final result? From fleet-server it shouldn't, I'm not sure about the package-registry-distribution. 
@mrodm WDYT?

Reply:
It shouldn't change the final result.
Once the tag is pushed, it can be triggered the package-registry-distribution job to create the new docker images.
@jsoriano do you see any issue here ?

Reply:
At the end triggering this pipeline in package-registry to create the docker images should be just tied to the event of pushing the new tag.
So it should be fine removing the `depends_on` section here @sharbuz